packages/calculator-bigint/etc/calculator-bigint.api.md

```diff
@@ -0,0 +1,62 @@
+## API Report File for "@dinero.js/calculator-bigint"

```

Comment:
I'm not sure these files should be in version control.

We should definitely publish a reference of the types, but since these files are generated automatically I would use them during documentation build to generate pages, then delete them. wdyt?

_Of course we won't handle the documentation generation in this PR, but it's the right moment to avoid versioning files we don't want._

Reply:
These files are intended to be used by maintainers to quickly see changes to external apis cause by new pull requests. If the file hasn't changed the api hasn't either.They recommend checking them into version control. Though, if you don't think they are helpful I can disable generating them.

Reply:
Also, there is a `temp` folder created in each package, it is not checked into version control, and a json file is generated in that folder that has all the data. It's used by the document generator, but you could use it how ever you like.

Reply:
Oh I just remembered! There is another important purpose to this file. Once everything is properly commented with `@public`, `@internal`, `@alpha`, etc. It will log all the errors into that file of where something is accidentally exposed publicly.

Reply:
That makes sense!